src/restapi/fHelp.ts

```diff
@@ -57,7 +56,7 @@ function addBullet(text: string): string {
 }
 
 function addBulletSlashCommand(text: string): string {
-    return `* \`/${AppID} ${text}\`\n`;
+    return `* \`/zendesk ${text}\`\n`;
```

Comment:
Maybe put this in a constant (commandTrigger) and use it also in the corresponding binding in the other PR?

Reply:
I agree with putting this in a `commandTrigger` constant.

Reply:
Good suggestion :+1: